.gitmodules

```diff
@@ -17,3 +14,7 @@
 [submodule "src/liblibc"]
 	path = src/liblibc
 	url = https://github.com/rust-lang/libc.git
+[submodule "src/libcompiler_builtins"]
+	path = src/libcompiler_builtins
+	url = git://github.com/japaric/rustc-builtins.git
```

Comment:
Before we land this I think we'll want to be sure to move this to rust-lang-nursery under the name "compiler-builtins"